applications/asset_tracker/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ set(PM_STATIC_YML_FILE
   ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml
   )
 
+list(APPEND mcuboot_OVERLAY_CONFIG
```

Comment:
@tejlmand Just a thought for a feature:

What if we supported adding overlay configs for images like it is for boards.

For this case we would have

`applications/asset_tracker/images/mcuboot.conf`

Which would imply that whatever is inside that file would be applied as an overlay to the mcuboot image.

Any thoughts?

Reply:
You mean like this proposal: https://github.com/nrfconnect/sdk-nrf/pull/3220#pullrequestreview-515598121
> If there is a need for automatic inclusion of overlays for child images, in a principle similar to board overlays, then please file a feature request, so this can be fixed in a proper way.

Reply:
Not exactly. In #3220 they explicitly checked for the existence of a file with a current name.

My proposal is that any overlay placed in `samples/hello_world/images/**child_image_name**.conf`  is applied to that child image.

The same way as it is today with boards names used for selecting devicetree overlay and kconfig automatically.